SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -97,6 +97,29 @@ FNetworkGUID USpatialPackageMapClient::GetNetGUIDFromEntityId(const Worker_Entit
 	return GetNetGUIDFromUnrealObjectRef(ObjectRef);
 }
 
+UObject* USpatialPackageMapClient::GetObjectFromUnrealObjectRef(const FUnrealObjectRef& ObjectRef)
+{
+	FNetworkGUID NetGUID = GetNetGUIDFromUnrealObjectRef(ObjectRef);
+	if (NetGUID.IsValid())
```

Comment:
Is it worth checking `!NetGUID.IsDefault()`?

Reply:
Sure